src/main/java/edu/rpi/legup/puzzle/nurikabe/rules/SurroundRegionDirectRule.java

```diff
@@ -3,14 +3,23 @@
 import edu.rpi.legup.model.gameboard.Board;
 import edu.rpi.legup.model.gameboard.PuzzleElement;
 import edu.rpi.legup.model.rules.DirectRule;
-import edu.rpi.legup.model.rules.ContradictionRule;
 import edu.rpi.legup.model.tree.TreeNode;
 import edu.rpi.legup.model.tree.TreeTransition;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeBoard;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeCell;
 import edu.rpi.legup.puzzle.nurikabe.NurikabeType;
+import edu.rpi.legup.puzzle.nurikabe.NurikabeUtilities;
+import edu.rpi.legup.utility.DisjointSets;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;

```

Comment:
Please avoid importing modules that are never used.

Reply:
Sorry just realized `Set` is actually being used, but this standard should still be upheld for future changes.